packages/shared/components/MenuLogin/index.ts

```diff
@@ -14,6 +14,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import MenuSshLogin from './MenuSshLogin';
-export default MenuSshLogin;
+import MenuLogin from './MenuLogin';
+export default MenuLogin;
```

Comment:
Some time ago we decided to prefer named export over default, this is a great opportunity to change it here